src/hooks/useNfc.ts

```diff
@@ -114,15 +135,31 @@ export const useNfc = () => {
               tag: "withdrawRequest";
               callback: string;
               k1: string;
+              pinLimit: number;
             }>(cardData);
 
+            let pin = "";
+            if (cardDataResponse.pinLimit !== undefined) {
```

Comment:
I was thinking that the pinLimit could be set to 0. But then one could argue that means it is "disabled" since the number in the pinLimit is the min amounted needed before asking the pin and the smallest possible number for that would be 1.
How should we implement it?

Reply:
Interesting. The doc says `value must be a positive integer (including 0)`, but also `to withdraw an amount equal to or greater [...]`.
But as we can't really withdraw 0 sats, I agree with you that 0 could means "disabled".
As this is not clear, you can keep it as `cardDataResponse.pinLimit !== undefined` and ignore my initial comment.